.circleci/config.yml

```diff
@@ -258,8 +258,7 @@ jobs:
           command: ./gradlew updateIntellijVersions
       - create_pull_request_if_files_changed:
           branch_name_prefix: update-intellij-versions
-          # Skip the first line with timestamp when calculating hash
```

Comment:
Now properties are saved without that annoying timestamp

Reply:
nice